libcontainer/cgroups/fs/cpuset.go

```diff
@@ -14,6 +16,41 @@ import (
 	"github.com/opencontainers/runc/libcontainer/configs"
 )
 
+var (
+	cpusetLock   sync.Mutex
+	cpusetPrefix string
+	cpusetInit   bool
+)
+
+func getCpusetPrefix() string {
+	cpusetLock.Lock()
+	defer cpusetLock.Unlock()
+
+	if cpusetInit {
+		return cpusetPrefix
+	}
```

Comment:
Better to use `sync.OnceValue for this kind of stuff. 

E.g.
```go
var getCpusetPrefix = sync.OnceValue(func() string{
....
})

Reply:
It is not applicable for new logic.